mazerunner/src/main/java/com/bugsnag/mazerunner/scenarios/ManualSessionScenario.java

```diff
@@ -0,0 +1,24 @@
+package com.bugsnag.mazerunner.scenarios;
+
+import com.bugsnag.Report;
+import com.bugsnag.callbacks.Callback;
+import com.bugsnag.mazerunner.Scenario;
+
+/**
+ * Sends a manual session payload to Bugsnag.
+ */
+public class ManualSessionScenario extends Scenario {
+    @Override
+    public void run() {
+        bugsnag.setAppVersion("1.2.3");
+        bugsnag.startSession();
+
+        try {
+            Thread.sleep(1000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
```

Comment:
Why is this delay necessary? This is not required in the android scenario.

Reply:
removed